l10n_mx_edi_qualtop/models/account_invoice.py

```diff
@@ -39,3 +39,10 @@ def invoice_validate(self):
             })
             inv.l10n_mx_edi_cfdi_name = inv.l10n_mx_edi_cfdi_second_name
         return result
+
+    @api.multi
+    def _l10n_mx_edi_create_cfdi_values(self):
```

Comment:
@GavyMG 

why not in the template?

Regards

Reply:
@luistorresm
I think it's better with the method (without adding a new file, etc) and ...

![image](https://user-images.githubusercontent.com/19410285/44230106-27ba6d80-a168-11e8-849d-8419058d9f6f.png)